eth/executionclient/config.go

```diff
@@ -10,4 +10,5 @@ import (
 type ExecutionOptions struct {
 	Addr              string        `yaml:"ETH1Addr" env:"ETH_1_ADDR" env-required:"true" env-description:"Execution client WebSocket address"`
 	ConnectionTimeout time.Duration `yaml:"ETH1ConnectionTimeout" env:"ETH_1_CONNECTION_TIMEOUT" env-default:"10s" env-description:"Execution client connection timeout"`
+	SyncTolerance     uint64        `yaml:"ETH1SyncTolerance" env:"ETH_1_SYNC_TOLERANCE" env-required:"false" env-default:"2" env-description:"The numbers of out-of-sync blocks we can tolerate"`
```

Comment:
@MatusKysel please have a look at the namings - if the follow convention and/or are proper (defaults, required)